### PR TITLE
Update phoenix_html version

### LINKED ISSUE #8: Outdated Dependencies
Looks like #3 is happening again:

```
Failed to use "phoenix_html" (versions 3.0.0 to 3.0.3) because
  ecto_interval (version 0.2.3) requires ~> 2.12
```

### PR DESCRIPTION
**Pull request by nathanl**
Closes #8

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -45,7 +45,7 @@ defmodule EctoInterval.Mixfile do
       {:ex_doc, ">= 0.0.0", only: :dev},
       {:postgrex, "~> 0.14.0 or ~> 0.15.0", optional: true},
       {:ecto, "~> 3.0", optional: true},
-      {:phoenix_html, "~> 2.12", optional: true}
+      {:phoenix_html, "~> 3.0", optional: true}
```
Comment (vheathen) [inline mix.exs:48]: Could you please change the phoenix_html required version to `"~> 2.12 or ~> 3.0"` to keep support with legacy projects?

Current projects probably have `{:ecto_interval, "~> 0.2.2"}` in `mix.exs` so automatic upgrade will effectively break their builds.

**Comment by vheathen** [comment]
@OvermindDL1 I would be very grateful if this PR can be merged

**Comment by OvermindDL1** [comment]
I have no clue how I didn't see a notification for this!  Pulled.

**Comment by OvermindDL1** [comment]
Tested, fixed docs because apparently something in ex_doc or elixir broke in a backwards incompatible way, and published to 0.2.4.  Thanks!

**Pull request merged**